components/core/ScenePageHeader.js

```diff
@@ -22,7 +21,7 @@ const STYLES_LEFT = css`
 
 const STYLES_RIGHT = css`
   flex-shrink: 0;
-  padding-left: 24px;
+  justify-self: end;
```

Comment:
You still want `padding-left` because when an element is right next to the other you don't want it to be right next to it.